### DISCUSSION TITLE
Allow creating new dashboards as default empty

### DISCUSSION THREAD

**Discussion by alexyao2015** [home-assistant/frontend#12917] [Dashboard]
I have so many entities in home assistant that when I go to create a new dashboard, my computer lags for several seconds until I can get into the menu to convert it into an empty dashboard.

A simple toggle to start new dashboards as empty would be great to fix this issue.

**Comment by ildar170975** [comment]
IMHO auto filling a new dashboard with entities is a useless functionality...
Probably auto filling new dashboards is needed for beginners only.
My proposal is - DO NOT fill them if a user set this switch to ON:
![image](https://user-images.githubusercontent.com/71872483/172734031-7e83abad-ce08-4648-bff3-1e7778bd2c95.png)
